pom.xml

```diff
@@ -98,6 +98,11 @@
                 <artifactId>script-security</artifactId>
                 <version>1.16</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
where it is used?

Reply:
This is required to set an upper bound for this dependency. Otherwise the [Maven Enforcer Plugin](https://maven.apache.org/enforcer/maven-enforcer-plugin/) will fail the build due to different versions of this dependency requested by various other dependencies. You can check this by removing these lines. Without them you can also look at the different versions required with `mvn dependency:tree`.